Yoast/javascript
# Transfer data from French stemmer to data file

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Transfers data from French stemmer to data file and improves stemming of words in -issement in French.
* [Yoast SEO Premium] Improves keyphrase recognition in French when the keyphrase contains words in -issement(s).

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Run yarn test and make sure that all tests pass.
* In French Premium: Try using different forms of words in the keyphrase and in the text/title/meta description, of which one ends in -issement. E.g., use forms like "accomplissement", "accompli", "accomplie", "accomplies", "accomplir", "accomplirait", "accomplis", "accomplissaient", "accomplissais", "accomplissait", "accomplissant", "accomplissent", "accomplit".

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-387
